### PR TITLE
Docs other fixes

### PR DESCRIPTION
**Pull request by praveenraj2018**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnavar/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/rnavar _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by praveenraj2018** [comment]
@maxulysse - Can you please check why the CI test fails on github. All CI tests runs successfully on my local with `pytest --tag <tag_name> --kwdof`

**Comment by maxulysse** [comment]
You're still using the old syntax for the config modules.
The recent modules updates got rid of the whole params.modules system, so you basically do everything with task.ext now.
Check out sarek, or rnaseq for an up-to-date syntax.

For the CI, I have the same issue as Github, so I'm guessing that the publishDir is not properly set, but I'd recommend looking out for the new syntax because it changes that too

File: .nf-core.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 lint:
   files_unchanged:
     - .github/workflows/linting.yml
+    - .github/CONTRIBUTING.md
```
Comment (maxulysse) [inline .nf-core.yml:4]: Why adding this file here?

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
Will need to fix lint in a separate PR.
But better to wait for the soon to arrive tools release

**Comment by praveenraj2018** [comment]
> Will need to fix lint in a separate PR. But better to wait for the soon to arrive tools release

Thanks!

**Pull request merged**